.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore temporary files and directories
+.ci-temp/
+.DS_Store
+
```

Comment:
Why there is no "build" folder ?
Do you have it on your local after Gradle build execution ?